core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -1392,7 +1399,10 @@ public synchronized void run() {
             }
             status = new Running();
             try {
-                Jenkins.getInstance().safeRestart();
+                // safeRestart records the current authentication for the log, so set it to the managing user
+                try (ACLContext _ = ACL.as(User.get(authentication, false, Collections.emptyMap()))) {
```

Comment:
I'm not sure it's going to work correctly if the User#get() method returns null.
May happen if a user gets deleted by somebody when hist session is active.

🐜